src/core/reopt_inputs.jl

```diff
@@ -190,14 +190,15 @@ function REoptInputs(s::AbstractScenario)
     months = 1:12
 
     levelization_factor, pwf_e, pwf_om, pwf_fuel, pwf_emissions_cost, pwf_grid_emissions, third_party_factor, pwf_offtaker, pwf_owner = setup_present_worth_factors(s, techs)
+    print("\nlevelization_factor: ", levelization_factor)
     # the following hardcoded values for levelization_factor matches the public REopt API value
     # and makes the test values match.
     # the REopt code herein uses the Desktop method for levelization_factor, which is more accurate
     # (Desktop has non-linear degradation vs. linear degradation in API)
     # levelization_factor = Dict("PV" => 0.9539)
     # levelization_factor = Dict("ground" => 0.942238, "roof_east" => 0.942238, "roof_west" => 0.942238)
     # levelization_factor["PV"] = 0.9539
-    # levelization_factor["Generator"] = 1.0
+    # levelization_factor["Generator"] = 1.0 # TODO: does it make sense to default this to 1 while the calculation involves discount and escalation rates?
```

Comment:
Not sure if you brought this up elsewhere, but seems like something to make an issue for if you think it's wrong. I'm not that familiar with the discount/escalation part of the "calculation", but I definitely thought 1 was appropriate for all techs that have have non-degrading production. I thought the PWFs accounts for discount/escalation of any cost-terms.

Reply:
I've since removed this comment. I also thought the PWFs accounted for discount/escalation rates but it seems like they're also baked into the levelization factors. Dan said he worked with Nick on this so I can follow up with him